docs/enriching-your-data/available-enrichments/cross-navigation-enrichment/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Cross Navigation Enrichment"
+sidebar_position: 15
+sidebar_label: Cross Navigation
+---
+
+This enrichment parses the extended cross navigation format in `_sp` querystring parameter and attach the cross_navigation context to an event.
+
+The extended cross navigation format can be described by `_sp={domainUserId}.{timestamp}.{sessionId}.{subjectUserId}.{sourceId}.{platform}.{reason}`
```

Comment:
@adatzer could you suggest a few more sentences we can say about this to explain why the enrichment is needed to a general audience?

Reply:
I've added [Matus's suggestion](https://github.com/snowplow/engineering-announcements/pull/106#discussion_r1494746906). Let me know if you want to add anything other.